src/resources/business-config.ts

```diff
@@ -80,7 +80,8 @@ export const ConfigJson = [
       act: 'Business Corporations',
       modalTitle: 'Voluntary Dissolution',
       confirmButtonText: 'Continue with Voluntary Dissolution',
-      dissolutionType: 'voluntarily dissolve'
+      dissolutionType: 'voluntarily dissolve',
+      incorporationText: 'an incorporated'
```

Comment:
I _think_ `incorporationText` and `subTitle` are only used in one place, and always together. Why not combine them?

Reply:
combined